tests/test_spstats.py

```diff
@@ -254,3 +254,96 @@ def percentile_80(a):
         # dataframe should contain three 1D binning of length 10 and three 2D binning of length 100 and one 2D binning of length 1000
         assert df.shape[0] == (1000 + 3 * 100 + 3 * 10)
 
+
```

Comment:
THERE ARE TWO LINES HERE WHEN THERE SHOULD BE ONLY ONE. REJECTED

Reply:
Haha, so I already made a mistake before even writing the first line

Reply:
The blank lines are the only bad part of your code!